thetis/options.py

```diff
@@ -119,14 +135,24 @@ class ExplicitTimestepperOptions2d(ExplicitTimestepperOptions):
         'sub_pc_type': 'ilu',
         'mat_type': 'aij',
     }).tag(config=True)
-    solver_parameters_tracer = PETScSolverParameters({
+
+
+class ExplicitTracerTimestepperOptions2d(ExplicitTimestepperOptions2d):
```

Comment:
Should these be subclassed from `ExplicitTimestepperOptions2d` and not directly from `ExplicitTimestepperOptions`. Of course it doesn't matter at the moment, but if this sets the ts options for tracer, and `ExplicitTimestepperOptions2d` for the swe then they are in my mind created equal - and if in the future we need to add an option that is specific to swe, we don't necessarily want to inherit it.